Packs/Okta_v2/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -0,0 +1,1007 @@
+            method="GET",
+            url_suffix=uri,
+            resp_type='response',
+            params=query_param
+        )
+        paged_results = response.json()
+        while "next" in response.links and len(response.json()) > 0:
+            next_page = response.links.get("next").get("url")
+            response = self._http_request(
+                method="GET",
+                full_url=next_page,
+                url_suffix='',
+                resp_type='response',
+                params=query_param
+
+            )
+            paged_results += response.json()
+        return paged_results
+
+    def get_group_members(self, group_id, limit):
+        uri = f'groups/{group_id}/users'
+        if limit:
+            uri += f'?limit={limit}'
+            return self._http_request(
+                method="GET",
+                url_suffix=uri
+            )
+        return self.get_paged_results(uri)
+
+    def list_groups(self, args):
+        # Base url - if none of the the above specified - returns all the groups (default 200 items)
+        uri = f"groups"
+        query_params = {}
+        for key, value in args.items():
+            if key == 'query':
+                key = 'q'
+            query_params[key] = encode_string_results(value)
+        if args.get('limit'):
+            return self._http_request(
+                method='GET',
+                url_suffix=uri,
+                params=query_params
+            )
+        return self.get_paged_results(uri, query_params)
+
+    def get_logs(self, args):
+        uri = f'logs'
+        query_params = {}
+        for key, value in args.items():
+            if key == 'query':
+                key = 'q'
+            query_params[key] = encode_string_results(value)
+        if args.get('limit'):
+            return self._http_request(
+                method='GET',
+                url_suffix=uri,
+                params=query_params
+            )
+        return self.get_paged_results(uri, query_params)
+
+    def delete_user(self, user_term):
+        uri = f"users/{encode_string_results(user_term)}"
+        return self._http_request(
+            method="DELETE",
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def clear_user_sessions(self, user_id):
+        uri = f'users/{user_id}/sessions'
+        return self._http_request(
+            method='DELETE',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+
+def test_module(client, data_args):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    uri = 'users/me'
+    client._http_request(method='GET', url_suffix=uri)
+    return demisto.results('ok')
+    raise ValueError('Test OktaV2 integration failed')
+
+
+# TODO: Manage the response codes
```

Comment:
?

Reply:
already managed it and forgot to delete the comment